doc/site/articles/team-terminology.markdown

```diff
@@ -24,7 +24,8 @@ For modders coming from Supreme Commander, the *team* is what SupCom calls an "a
  * resources
  * colour
  * starting point
- * extra traits, such as faction (the engine does not natively recognize the concept of factions but it's easy to add them)
+ * faction (aka side)
+ * any other custom extra traits you add.
```

Comment:
I couldn't think of a good universal example of something that could be Lua'd which is obvious without too much explanation.

Reply:
Where is the concept of faction encoded in the engine?

Reply:
Yes, "side".